tools/ktlint

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
+
+MAG="\033[95m"
+END="\033[0m"
+
+status() {
+    echo -e "$MAG$1$END"
+}
+
+ROOT=$(dirname $0)/..
+cd $ROOT
+
+status "Running ktlint on src_kt/..."
+# TODO(wkorman): Consider moving ktlint to a sigh task and (as we do
+# on Travis currently) only ktlint'ing modified files.
+find ./src_kt -name "*.kt" | xargs ktlint --android --relative --verbose .
```

Comment:
Don't we want to lint code in //src/ and //particles/?

Reply:
We don't do this on Travis currently I believe, I added a note to the TODO.

Reply:
The travis script is running: `git diff --name-only --relative $travis_commit_range | grep '\.kt[s\"]\?$' | xargs ktlint --android --relative --verbose .` I thought this tested the other two source directories (it does filter by git diffs). 
The TODO makes me happy.